src/Evolution/Systems/GrMhd/ValenciaDivClean/BoundaryConditions/DirichletAnalytic.hpp

```diff
@@ -208,166 +142,37 @@ class DirichletAnalytic final : public BoundaryCondition {
                  domain::Tags::ElementMap<3, Frame::Grid>,
                  domain::CoordinateMaps::Tags::CoordinateMap<3, Frame::Grid,
                                                              Frame::Inertial>,
-                 fd::Tags::Reconstructor, ::Tags::AnalyticSolutionOrData>;
-
-  template <typename AnalyticSolutionOrData>
+                 fd::Tags::Reconstructor>;
   void fd_ghost(
-      const gsl::not_null<Scalar<DataVector>*> rest_mass_density,
-      const gsl::not_null<Scalar<DataVector>*> electron_fraction,
-      const gsl::not_null<Scalar<DataVector>*> temperature,
-      const gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*>
+      gsl::not_null<Scalar<DataVector>*> rest_mass_density,
+      gsl::not_null<Scalar<DataVector>*> electron_fraction,
+      gsl::not_null<Scalar<DataVector>*> temperature,
+      gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*>
           lorentz_factor_times_spatial_velocity,
-      const gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*>
-          magnetic_field,
-      const gsl::not_null<Scalar<DataVector>*> divergence_cleaning_field,
+      gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*> magnetic_field,
+      gsl::not_null<Scalar<DataVector>*> divergence_cleaning_field,
 
-      const gsl::not_null<std::optional<Variables<db::wrap_tags_in<
+      gsl::not_null<std::optional<Variables<db::wrap_tags_in<
           Flux, typename grmhd::ValenciaDivClean::System::flux_variables>>>*>
           cell_centered_ghost_fluxes,
 
       const Direction<3>& direction,
 
       // fd_interior_temporary_tags
-      const Mesh<3> subcell_mesh,
+      Mesh<3> subcell_mesh,
```

Comment:
I think `const` stays here?

Reply:
This is a clang-tidy thing, the `const` has no effect in the header file so clang-tidy says to remove it

Reply:
Yeah if it's not a reference, then no `const` in hpp. However, it looks like it's const in the `cpp`, so can this not just be a `const&`?

Reply:
`const Mesh<3>& subcell_mesh` ?